### PR TITLE
Added: Pagination to the 'aws' Provider reader

### LINKED ISSUE #13: AWS: Support pagination
Right now we do not iterate over the results from AWS, we just use the first page, we should read all the pages of each resource to have the full list of services for that resource.

Related: https://github.com/cycloidio/terracognita/issues/52

### PR DESCRIPTION
**Pull request by xescugc**
This PR implements the pagination on the AWS provider.

Closes #13

### CODE REVIEW

File: aws/cmd/template.go [hunk 1]
```diff
@@ -139,6 +178,25 @@ type Function struct {
 	// FilterByOwner adds the "{{.FilterByOwner}} = AccountID" to the input filter
 	// so this value has to be the correct name on the input
 	FilterByOwner string
+
+	// HasPagination flags if the resource has NextToken logic or not
```
[the remark this answers is not in the document]
Comment (xescugc) [reply aws/cmd/template.go]: Fixed that :)

File: go.mod [hunk 2]
```diff
@@ -9,13 +9,17 @@ require (
 	github.com/aws/aws-sdk-go v1.25.4
 	github.com/chr4/pwgen v1.1.0
 	github.com/cycloidio/tfdocs v0.0.0-20200111145532-e6a80a93d7cc
+	github.com/davecgh/go-spew v1.1.1
```
Comment (talset) [inline go.mod:12]: Should it be removed ?

Comment (xescugc) [reply go.mod:12]: ou yep haha

**Comment by xescugc** [comment]
Ok i've ran `go mod tidy` to tidy things with the vendors. The tests also passed just to test it.

**Comment by xescugc** [comment]
Ok all the things are fixed and the test pass again :)

**Comment by tormath1** [comment]
> Ok all the things are fixed and the test pass again :)

and it's ok for me. you can RS and merge :)

**Pull request merged**